index.html

```diff
@@ -87,6 +88,33 @@ <h2 id="rules-wd">
           &lt;button role="heading"&gt;search&lt;/button&gt;
         </pre>
       </aside>
+
+      <aside class="example">
+        The following uses a `role=group` on a `fieldset` element. This is unnecessary,
+        because the `fieldset` element is implicitly exposed as a `role=group`.
+        <pre class="HTML">
+          &lt;fieldset role="group"&gt;...&lt;/fieldset&gt;
+        </pre>
+      </aside>
+
+      <aside class="example">
+        The following uses a `role=list` on an `ul` element. This is generally unnecessary,
+        because the `ul` element is implicitly exposed as a `role=list`. However, some user agents suppress a list's implicit ARIA semantics if list markers are removed. Authors can use `role=list` to reinstate the role if necessary, though this practice would generally not be recommended.
+        <pre class="HTML">
+          &lt;ul role="list"&gt;...&lt;/ul&gt;
+        </pre>
+      </aside>
+
+      <aside class="example">
+        The following uses a `role=button` on a `summary` element. This is unnecessary and can cause cross-platform issues. Setting `role=button` to the `summary` element can actually prevent correctly exposing this element to assistive technologies.
```

Comment:
"prevent correctly exposing this element" > "prevent this element from being correctly exposed"